app/controllers/application_controller.rb

```diff
@@ -25,4 +25,8 @@ def current_user
   def user_signed_in?
     current_user.persisted?
   end
+
+  def on_member?
```

Comment:
how should i recognize what action to proceed in controller? there should be a diffrence between member and collection response